tokens.xml

```diff
@@ -6153,6 +6272,21 @@ Equip {1}</text>
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Skeleton </name>
+            <prop>
+                <colors>B</colors>
+                <type>Token Creature — Skeleton</type>
+                <maintype>Creature</maintype>
+                <cmc>0</cmc>
+                <pt>1/1</pt>
+            </prop>
+            <set picURL="https://c1.scryfall.com/file/scryfall-cards/large/front/f/a/fa6fdb57-82f3-4695-b1fa-1f301ea4ef83.jpg?1626139846">AFR</set>
+            <reverse-related>Skeletal Swarming</reverse-related>
```

Comment:
Death-Priest of Myrkul creates a skeleton.

Reply:
Added